pom.xml

```diff
@@ -31,7 +31,16 @@
             <artifactId>testng</artifactId>
             <version>6.11</version>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-engine</artifactId>
+            <version>5.4.0</version>
```

Comment:
Move all versions of dependencies into <properties> section. It helps to manage the same versions for different libraries (f.e. junit-jupiter-engine and junit-jupiter-params) and generally is the best practice